commons/src/main/java/io/hyscale/commons/utils/ThreadPoolUtil.java

```diff
@@ -53,7 +53,7 @@ public boolean execute(Runnable runnable) {
 			executor.execute(runnable);
 			return true;
 		} catch (RejectedExecutionException re) {
-
```

Comment:
Instead of comments, we can add log statement here, to get a better idea. Same with other exceptions in this class.